airflow/api_connexion/endpoints/task_instance_endpoint.py

```diff
@@ -522,19 +522,10 @@ def post_set_task_instances_state(*, dag_id: str, session: Session = NEW_SESSION
     if not task:
         error_message = f"Task ID {task_id} not found"
         raise NotFound(error_message)
-
-    logical_date = data.get("logical_date")
```

Comment:
`logical_date` should also be removed from `SetTaskInstanceStateFormSchema` (where this key came from) and the API spec.

Reply:
removed it